src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/DwpState.java

```diff
@@ -32,7 +32,8 @@ public enum DwpState {
     PHE_REFUSED("phmeRefused", "PHE refused"),
     DECISION_ACTIONED("decisionActioned", "Decision actioned"),
     HEARING_RECORDING_PROCESSED("hearingRecordingProcessed","Hearing recording processed"),
-    HEARING_RECORDING_ACTIONED("hearingRecordingActioned","Hearing recording actioned");
+    HEARING_RECORDING_ACTIONED("hearingRecordingActioned","Hearing recording actioned"),
+    HEARING_STATE_ISSUED("hearingStateIssued", "Hearing state issued");
```

Comment:
Is there a ccd definitions pr adding this state to the fixed list? 
If not can you do this otherwise you won't be able to use this value.

Reply:
Updated text from "state" to "date", to be in line with changes in ccd-def project. See https://github.com/hmcts/sscs-ccd-definitions/pull/1021 for Hurman's changes there.